docs/examples/101/basic-batch-account/main.bicep

```diff
@@ -0,0 +1,20 @@
+param BatchaccountName string
+param allocationMode string {
+  default: 'BatchService'
+  allowed: [
+    'BatchService'
+    'Usersubscription'
+  ]
+}
+
+param location string = resourceGroup().location
+
+resource batchaccount 'Microsoft.Batch/batchAccounts@2020-09-01' = {
+  name: BatchaccountName
+  location: location
+  properties: {
+    poolAllocationMode: allocationMode
+  }
+}
+
+output batchaccountFQDN string = batchaccount.properties.accountEndpoint
```

Comment:
You need a blank line after line 20 to pass the formatting test

Reply:
@alex-frankel , Tried adding the blank line but the cursor hits back at the end of line 20 as soon as I save the file. Could you please let me know if I am missing onto something basic? Thank you.

Reply:
You might have this setting enabled:
![image](https://user-images.githubusercontent.com/11078275/103174130-38457d00-4825-11eb-93bc-38008bd3dbc5.png)